### ISSUE TITLE
Use HTTPS

### ISSUE THREAD

**Issue by dzucconi** [artsy/folio.artsy.net#12]

**Comment by dylanfareed** [comment]
In your estimation @jessekedy is this worth 20 bucks/month?

**Comment by craigspaeth** [comment]
:stuck_out_tongue_winking_eye: :fire:Heroku SSL addon:fire:. FWIW I think this static site is deployed to S3 so it shouldn't have that price tag.

**Comment by dzucconi** [comment]
Indeed, it lives here: https://s3.amazonaws.com/folio.artsy.net/index.html

**Comment by jessekedy** [comment]
Probably. Would it be easier to recreate the page in Marketo?

**Comment by dylanfareed** [comment]
Tight. I'll set up the DNS and whatever else via Cloudfront. Thanks @dzucconi
